app/Http/Controllers/User/NotificationController.php

```diff
@@ -19,32 +19,34 @@ public function __construct(
         $this->notification = $notification;
     }
 
-    public function getNotifications($limit = -1)
+    public function getNotifications()
     {
         $with = [
             'target',
             'userSend',
         ];
         $where = [
             'receive_id' => Auth::id(),
-        ];
-
-        return $data = $this->notification->getNotifications($limit, $with, $where);
+        ];       
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Controllers/User/NotificationController.php</br>**Line**: 30</br>Whitespace found at end of line